File: Python Code/Daily_Flash_Code_0/Week9/19Mar/51-DailyFlash_Solutions/19-Mar/Python/prog3.py
import array
num = int(input())
rev = 0
while(num != 0):
    rev = rev * 10 + num % 10
    num = int(num / 10)
arr = array.array("i", [])
while(rev != 0):
    arr.append(rev % 10)
    rev = int(rev / 10)
for i in arr:
    print(i, end = " ")

File: Python Code/Daily_Flash_Code_0/Week9/19Mar/51-DailyFlash_Solutions/19-Mar/C++/prog3.cpp
#include<iostream>

int main() {
	int num;
	std::cin >>num;
	int rev = 0, n = 0;
	while(num != 0) {
		rev = rev * 10 + num % 10;
		num /= 10;
		n++;
	}
	int arr[n];
	int i = 0;
	while(rev != 0) {
		arr[i] = rev % 10;
		rev /= 10;
		i++;
	}
	for(int i = 0; i < n; i++) {
		std::cout << arr[i];
	}

}
